### ISSUE TITLE
Tab bar location

### ISSUE THREAD

**Issue by dalek1967** [yfdyh000/tabutils#47]
In preferences under Appearance then the Tab Bar tab, I select for the tabs to be on the left for Tab bar features but they move to the right instead.  If I select right, they are still on the right.  Top and bottom work just fine.  I just can't get them on the left side.  This used to work a long time ago, about a year or two ago.  Is it possible to correct this when you get time?  It's more of a annoyance than anything but being able to have them on the left in certain profiles would be nice.  

Thanks much.

**Comment by yfdyh000** [comment]
This is a strange situation, I have not encountered it.

Can you reproduce it using a clean profile?

**Comment by dalek1967** [comment]
I only tried this in the one profile I want the tabs on the left.  Naturally, it doesn't work there but it does work in the others when I tested it.  I guess I have some sort of conflict going on with that one profile somewhere.  That could be fun to figure out.  

I've also been trying to figure out how to make that little x show on all tabs for closing it.  While playing with that, now even the little lock doesn't show up for when a tab is protected.  Oh, still no little x for closing the tab either.  

I guess my issue is some setting here so we'll close this one.  

Thanks for the neat add-on.
